index.js

```diff
@@ -34,6 +33,29 @@ function load() {
   return result;
 }
 
+function extend(a, b) {
+  // iterate over all direct and inherited enumerable properties
```

Comment:
Thank you for all of these thoughtful comments and questions. I also agree so much!

This leniency was done purposefully, because that’s how it behaved in the old (jQuery) extend method. From the [`extend` package documentation](https://www.npmjs.com/package/extend):

> ... properties inherited from the object's prototype will be copied over.

This is why I added the explicit comment. At least now it’s out in the open? 😅

Reply:
Yes, very good to shine a light on it. It made me nervous since in BCD we have properties like "toString" and "hasOwnProperty" where `p in {}` would return true. It seems like we don't have any accidents because of it, but dealing only with enumerable own properties seems preferable, or really matching whatever the behavior of `Object.assign` is. (I'm still not sure about the case where there's an enumerable own property on the one side and a non-enumerable own property on the other. We can't have that in BCD, though.)